Tests/UnitTests/SubscriberAttributes/__Snapshots__/BackendSubscriberAttributesTests/iOS14-testPostReceiptWithSubscriberAttributesPassesErrorIfStatusCodeIsNotSuccess.1.json

```diff
@@ -0,0 +1,29 @@
+{
+  "headers" : {
```

Comment:
Just to make sure, these files are snapshots of the requests done during the unit tests. Then the tests check that the request remains the same, correct? This is nice!

Reply:
Yup exactly :) without needing to check each thing individually.